authority/config.go

```diff
@@ -46,6 +47,7 @@ type Config struct {
 	DNSNames         []string            `json:"dnsNames"`
 	Logger           json.RawMessage     `json:"logger,omitempty"`
 	DB               *db.Config          `json:"db,omitempty"`
+	ACME             *acme.Config        `json:"acme,omitempty"`
```

Comment:
It should be a provisioner.

Reply:
Let's chat about this when you feel better. I was under the impression that acme provisioners would go into the existing provisioner list. As I'm thinking about them right now, we should be able to have many acme provisioners, each associated with it's own acme account.